bin/node-sass

```diff
@@ -204,6 +246,15 @@ function run(options, emitter) {
     options.includePath = [options.includePath];
   }
 
+  if(isDirectory(options.src)) {
+    if (!options.output) {
+      emitter.emit('error', 'Specify an output directory when using multi-file compilation');
+    }
+    if (!isDirectory(options.output)) {
+      emitter.emit('error', ['Multi-file compilation: requires destination', options.output, 'to be a directory.'].join(' '));
```

Comment:
:+1:
We should refactor the code with formatting error. I just followed @kevva's lead on that one :stuck_out_tongue_winking_eye: 
(we have used that array-join thingy a lot -- all over the place!)